CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ Note that 3.1.0 will be the first API stable release and interfaces in this rele
 
 * Add top-level `CMakeLists.txt`
   ([\#1475](https://github.com/seqan/seqan3/pull/1475)).
+* We now support Doxygen up to version 1.8.17.
```

Comment:
```suggestion
* We now support Doxygen up to version 1.8.17
```

(The period of the sentence is after the PR link)

Reply:
Maybe also more like

```suggestion
* We now use Doxygen version 1.8.17 to build our documentation
```

We can't really guarantee `up to` (probably) because many things break inbetween versions.
`support` is debateable but I think "we use doxygen-1.8.17 to build the docs" captures the spirit of the change more?

Reply:
That makes sense, I changed the entry again.